### PR TITLE
Branch server configuration bits

### PR DESCRIPTION
**Pull request by aaannz**
After branch server configuration highstate needs to be run. Then before terminal can be booted, images need to be synchronized to the branch server.

### CODE REVIEW

**Comment by aaannz** [comment]
Partly fixes #6067, screenshots are missing

**Comment by aaannz** [comment]
Partly fixes https://github.com/SUSE/spacewalk/issues/6068, does not provide instructions to check if terminal is using proxy or not

**Comment by Loquacity** [comment]
Hi @aaannz I am not comfortable with these large-scale changes to the doc in the last couple of days of reviews. Can we merge these for the next release please, so that we have time to go over them? If there are small changes that you feel are critical, then let me know.

File: adoc/book_retail_getting_started.adoc [hunk 1]
```diff
@@ -11,4 +11,6 @@ include::retail_chap_install.adoc[leveloffset=1]
 
 include::retail_chap_admin.adoc[leveloffset=1]
 
+include::retail_salt_formulas.adoc[leveloffset=1]
```
Comment (Loquacity) [inline adoc/book_retail_getting_started.adoc]: This document hasn't been reviewed.

**Comment by aaannz** [comment]
I added formulas file (which was present before, only not enabled) because of https://github.com/SUSE/spacewalk/issues/6067. I can split this to different PR. What I would like to see merged is the information about image synchronization.

**Comment by Loquacity** [comment]
Let's re-address after initial release.

**Pull request closed without merging**